README.md

```diff
@@ -1 +1,131 @@
-# ckeditor-plugin
+# ImageKit CKEditor 5 Plugin
+
+This plugin integrates ImageKit into a custom CKEditor 5 build, allowing you to use the embedded Media Library Widget from within your CKEditor UI. 
+
+With this plugin, you can directly insert images from your ImageKit account into the editor, as well as upload new images to your Media Library.
+
+<img src="https://gblobscdn.gitbook.com/assets%2F-Lulz44snZCQCyuqQ_vg%2F-MTKIls8r82OVCOK9Ouz%2F-MTKJBxZUiN_G6sSysBN%2Fckeditor-4.png?alt=media&token=ed08a3e0-2912-4d8e-a5b5-5ffdfb85a3c6" alt="Image insertion" />
```

Comment:
host assets in an assets folder in same repo and refer that like we have done in uppy uploader plugin for gif.

Reply:
Done